probe-rs/targets/GD32F10x_Series.yaml

```diff
@@ -18,8 +18,7 @@ variants:
     cores:
     - main
     access:
-      read: true
-      write: true
+      execute: false
```

Comment:
So my (deleted) comment was right, after all? If the pack marks RAM as !executable, we will need to patch these by hand unfortunately. If no RAM is executable, probe-rs will not be able to run the flash algo, and so it won't be able to flash these MCUs.